src/common/options.cc

```diff
@@ -5601,6 +5601,18 @@ std::vector<Option> get_global_options() {
     Option("debug_heartbeat_testing_span", Option::TYPE_INT, Option::LEVEL_DEV)
     .set_default(0)
     .set_description("Override 60 second periods for testing only"),
+
+    Option("librados_thread_count", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
```

Comment:
these options should be added to the docs or made dev options. I'd prefer the latter unless there's strong evidence that these will need adjustment by users, since we have too many options for users already